src/nerve/frontends/tui/commander/workflow_runner.py

```
@@ -0,0 +1,1028 @@
+        """Copy current step's input/output to clipboard."""
+        if not self.steps or self.selected_step_index >= len(self.steps):
+            return
+
+        step = self.steps[self.selected_step_index]
+        text = f"=== INPUT ===\n{step.input_text}\n\n=== OUTPUT ===\n{step.output_text}"
+
+        try:
+            import subprocess
+
+            process = subprocess.Popen(
+                ["pbcopy"], stdin=subprocess.PIPE, env={"LANG": "en_US.UTF-8"}
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)